### DISCUSSION TITLE
Is there a way to make Oppressor MK2 unavailable to all players in the session?

### DISCUSSION THREAD

**Discussion by iwanttodo1t** [YimMenu/YimMenu#3255] [General]
I think the Oppressor MK2 upsets the balance of the game and would probably be more fun without the vehicle.
i am session host.

**Comment by lonelybud** [comment]
Its their property they do whatever they want with it. You dont deserve to have fun alone. Don't ask silly questions again.

**Reply by iwanttodo1t** [reply]
Does Rockstar respect players?
Does the toxic feature respect players?

**Reply by Tayipc38** [reply]
> Its their property they do whatever they want with it. You dont deserve to have fun alone. Don't ask silly questions again. You sound like Choji Tomiyama in Wind Breaker anime. Go watch it.

Not only did you reference something off topic, you failed to grasp what kind of a project this repo is home to. Ill assume you were under some kind of influence when writing this, otherwise things aren't looking too bright for you.

**Reply by lonelybud** [reply]
he is asking to remove oppressor mk2 from whole session and he thinks its fun. Off topic stuff you are right I should not have mentioned that

Edit It is fine if he has some idea like auto disable someones pv who is targeting you or is going to grief you.

> things aren't looking too bright for you.

please provide some bright ideas.

**Reply by Tayipc38** [reply]
> he is asking to remove oppressor mk2 from whole session and he thinks its fun. Off topic stuff you are right I should not have mentioned that
> 
> Edit It is fine if he has some idea like auto disable someones pv who is targeting you or is going to grief you.
> 
> > things aren't looking too bright for you.
> 
> please provide some bright ideas.

Here is a bright idea, on a project that is dedicated to cheat in a game, has a big portfolio of griefing features, mentioning 'deserving fun' is a big sign of lack of awareness. Another bright idea would be to not be passive aggressive as it sinks you even further.

**Reply by lonelybud** [reply]
i think you got provoked regarding the anime stuff i get it. I edited my comment. Wont happen again alright!

**Comment by iwanttodo1t** [comment]
I already had a way to do it, but I didn't want to do it because of the possibility that the player might leave the session as a result

**Reply by Tayipc38** [reply]
I actually have a private script that works against non-modders. It systematically removes the gun attachments of weaponized vehicles. [vehicle_disarm.cpp](https://pastebin.com/vTJcNLnZ). Its a nice way of repelling them without causing them to accuse you or leave the session.

**Reply by iwanttodo1t** [reply]
> I actually have a private script that works against non-modders. It systematically removes the gun attachments of weaponized vehicles. [vehicle_disarm.cpp](https://pastebin.com/vTJcNLnZ). Its a nice way of repelling them without causing them to accuse you or leave the session.

Wow, thank you
